src/Components/Layout/Fonts.tsx

```diff
@@ -29,13 +40,13 @@ export const Fonts = (): JSX.Element => (
       @font-face {
         font-family: 'Inconsolata';
         font-style: normal;
-        font-weight: 700;
-        src: url('../../../Fonts/Inconsolata-Black.eot');
+        font-weight: 900;
+        src: url('../../../Fonts/inconsolata-v21-latin-900.eot');
         src:
-            url('../../../Fonts/Inconsolata-Black.eot?#iefix') format('embedded-opentype'),
-            url('../../../Fonts/Inconsolata-Black.woff2') format('woff2'),
-            url('../../../Fonts/Inconsolata-Black.woff') format('woff'),
-            url('../../../Fonts/Inconsolata-Black.ttf') format('truetype');
+            url('../../../Fonts/inconsolata-v21-latin-900.eot?#iefix') format('embedded-opentype'),
```

Comment:
Checking Cate's Figmas, it looks like the percentages on the landing page are supposed to be font weight 900, though not sure if this difference is noticeable